App/components/Bible/Verse.js

```diff
@@ -9,37 +9,41 @@ export default function Verse({ number, text }) {
   const size = useSelector(selectors.fontSize);
   const isEnglish = useSelector(selectors.language) === 'english';
 
-  if (!isEnglish)
-    return (
-      <VerseContainer size={size}>
-        <Text size={size}>
-          {number} {text}
-        </Text>
-      </VerseContainer>
-    );
-
   const isMounted = useIsMounted();
 
   const [verseParts, setVerseParts] = useState(breakVerseIntoParts());
   const [startsWithItalics, setStartsWithItalics] = useState(doesItStartWithItalics());
 
   useEffect(() => {
     if (!isMounted) return;
+    if (!isEnglish) return;
```

Comment:
Is this because we don't have any italics in the Chinese translations?

Reply:
Correct